### ISSUE TITLE
Case let in guard doesn't work correctly

### ISSUE THREAD

**Issue by brentleyjones** [nicklockwood/SwiftFormat#896]
In the following code:
```swift
guard
    let first = response?.first,
    let last = response?.last,
    case .cardToken(token: let firstToken ,let provider: firstProvider) = first,
    case .cardToken(token: let lastToken, let provider: lastProvider) = last
else {
    XCTFail("Should return at least two LowValueTokens")
    return
}
```

SwiftFomat says:
```
error: Expected { on line 4 in Example.swift.
```

but changing it to this fixes it:
```swift
guard
    let first = response?.first,
    let last = response?.last,
    case let .cardToken(token: firstToken, provider: firstProvider) = first,
    case let .cardToken(token: lastToken, provider: lastProvider) = last
else {
    XCTFail("Should return at least two LowValueTokens")
    return
}
```

**Comment by nicklockwood** [comment]
@brentleyjones I assume it's just a typo, but in the first example you've written:

```swift
case .cardToken(token: let firstToken, let provider: firstProvider) = first,
case .cardToken(token: let lastToken, let provider: lastProvider) = last
```

Which isn't valid syntax. I assume you meant this?

```swift
case .cardToken(token: let firstToken, provider: let firstProvider) = first,
case .cardToken(token: let lastToken, provider: let lastProvider) = last
```

**Comment by brentleyjones** [comment]
Sorry, yes.

**Comment by nicklockwood** [comment]
@brentleyjones fixed in 0.48.0
